learningequality/studio
# Somewhat optimize django page loading

## Description

* Does some small optimizations to space calculation
* Defers loading of space related (and token) data to increase page load times
* Cleans up unused Python code

#### Issue Addressed (if applicable)

Reduces page load bottleneck that was dependent on the DB

## Steps to Test

- Open the edit modal, confirm storage size shows
- Open settings page, confirm Account and Storage tabs load as expected
